### PR TITLE
[PLGN-612]- Set GRR Plugin to Obsolete

### PR DESCRIPTION
**Pull request by jerichardson-r7**
### Description

In order to remove the GRR plugin we need to set it to obsolete first.
https://issues.corp.rapid7.com/browse/PLGN-612

  -

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``rapid7/insightconnect-python-3-38-slim-plugin:{sdk-version-num}`` and ``rapid7/insightconnect-python-3-38-plugin:{sdk-version-num}``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``insight-plugin validate`` which calls ``icon_validate`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `insight-plugin samples`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `insight-plugin validate` and make sure everything passes
2. Run the assessment tool: `insight-plugin run -A`. For single action validation: `insight-plugin run tests/{file}.json -A`
3. Copy (`insight-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Review by igorski-r7 — APPROVED** [review]

**Review by dlaverty-r7 — APPROVED** [review]

**Comment by rbowden-r7** [comment]
there is the following dependabot prs against this plugin

https://github.com/rapid7/insightconnect-plugins/pull/2074
https://github.com/rapid7/insightconnect-plugins/pull/1905
https://github.com/rapid7/insightconnect-plugins/pull/1798

as well as there is still a few snyk vulns thats will not be resolved. in the uI as a result of making the plugin obsolete in this way 
https://app.snyk.io/org/icon-fwb/project/d015444c-5381-4244-a89b-04e176133f63

File: plugins/grr/help.md [hunk 1]
```diff
@@ -335,19 +335,23 @@ Example input:
...
 
 ## Troubleshooting
-
-_This plugin does not contain any troubleshooting information._
+  
+*There is no troubleshooting for this plugin.*
 
 # Version History
 
+* 3.0.1 - Updated plugin to set the status as obsolete
```
Comment (dlaverty-r7) [inline plugins/grr/help.md:354]: (Suggestion) -> Updated plugin to set the status to obsolete. This plugin is no longer supported.

Comment (jerichardson-r7) [reply plugins/grr/help.md]: Thanks 🙂  updated.

**Comment by jerichardson-r7** [comment]
> there is the following dependabot prs against this plugin
> 
> #2074 #1905 #1798
> 
> as well as there is still a few snyk vulns thats will not be resolved. in the uI as a result of making the plugin obsolete in this way https://app.snyk.io/org/icon-fwb/project/d015444c-5381-4244-a89b-04e176133f63

Thanks Robert 🙂 

So we need to mark this as obsolete first then we are going to remove it in a seperate PR which will resolve the synk vuls.

**Pull request merged**